src/main/target/STM32G47X/target.mk

```diff
@@ -1,2 +1,3 @@
 TARGET_MCU        := STM32G474xx
 TARGET_MCU_FAMILY := STM32G4
+HSE_VALUE         := 16000000
```

Comment:
This can be removed given: https://github.com/betaflight/betaflight/pull/12632

Reply:
Done.